src/theme/basic/index.scss

```diff
@@ -138,6 +155,73 @@ $group-summary: #007fa3;
         overflow: visible;
     }
 
+    .q-grid-icon-button[disabled][disabled] {
+        color: $icon-button-disabled;
+    }
+
+    .q-grid-pager {
+        font-family: Roboto, "Helvetica Neue", sans-serif;
+        padding: 11px 0;
+        height: 24px;
+        font-size: 12px;
+        color: $pager-primary;
+
+        .pager-select {
+            height: 32px;
+            padding: 3px 0;
+
+            select {
+                font-family: Roboto, "Helvetica Neue", sans-serif;
+                font-size: 12px;
+                color: $pager-primary;
+                border: none;
+                height: 24px;
+                width: 34px;
+                margin: 0 0 0 12px;
+            }
+            select:focus {
+                outline: none;
+            }
+        }
+
+        .q-grid-page {
```

Comment:
q-grid-pager-page?

Reply:
That class has been copied from the corresponding Material template. So, please, keep in mind that we have the same issue there as well.

But I will fix this anyway.

Reply:
done in my last commit